diff --git a/src/linux/gpu.rs b/src/linux/gpu.rs
index 515b78ed..3edd965b 100644
--- a/src/linux/gpu.rs
+++ b/src/linux/gpu.rs
@@ -82,6 +82,8 @@ pub fn gpu_jail(cfg: &Config, policy: &str) -> Result<Option<Minijail>> {
             jail_mount_bind_if_exists(
                 &mut jail,
                 &[
+                    "/run/opengl-driver",
+                    "/nix/store",
                     "/usr/lib",
                     "/usr/lib64",
                     "/lib",
