diff --git a/src/linux/gpu.rs b/src/linux/gpu.rs
index 3edd965b..ae4531eb 100644
--- a/src/linux/gpu.rs
+++ b/src/linux/gpu.rs
@@ -165,7 +165,7 @@ pub fn create_gpu_device(
     map_request: Arc<Mutex<Option<ExternalMapping>>>,
 ) -> DeviceResult {
     let mut display_backends = vec![
-        virtio::DisplayBackend::X(x_display),
+        // virtio::DisplayBackend::X(x_display),
         virtio::DisplayBackend::Stub,
     ];
 
@@ -176,12 +176,14 @@ pub fn create_gpu_device(
         .collect::<Option<Vec<_>>>()
         .ok_or_else(|| anyhow!("wayland socket path has no parent or file name"))?;
 
+    /*
     if let Some(socket_path) = wayland_socket_path {
         display_backends.insert(
             0,
             virtio::DisplayBackend::Wayland(Some(socket_path.to_owned())),
         );
     }
+    */
 
     let dev = virtio::Gpu::new(
         exit_evt.try_clone().context("failed to clone event")?,
